apps/aechannel/src/aesc_session_noise.erl

```diff
@@ -352,7 +352,7 @@ locate_fsm(Type, MInfo, #st{ init_op = {accept, SnInfo, _Opts} } = St) ->
     Info = Info0#{reestablish => (Type =:= ?CH_REESTABL)},
     Cands = get_cands(Type, Info),
     lager:debug("Cands = ~p", [Cands]),
-    try_cands(Cands, 3, 0, Type, Info, St).
+    try_cands(Cands, 5, 0, Type, Info, St).
```

Comment:
The retry logic should not use a static backoff but at least an exponential backoff. It should also be configurable. Both improvements should be handled separately though.